architecture/additional_concepts/authentication.adoc

```diff
@@ -88,6 +88,9 @@ the `system:anonymous` virtual user and the `system:unauthenticated` virtual
 group to the request. This allows the authorization layer to determine which
 requests, if any, an anonymous user is allowed to make.
 
+See link:../../rest_api/index.html[rest API documentation] for more information
```

Comment:
s/rest/REST/

Maybe:

See the link:../../rest_api/index.html[REST API Overview]